src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -60,7 +60,7 @@ public function __construct(ContainerBuilder $container, PhpFileLoader $loader,
         $this->container->loadFromExtension($namespace, static::processValue($config));
     }
 
-    final public function import(string $resource, string $type = null, bool $ignoreErrors = false)
+    final public function import(string $resource, string $type = null, int $ignoreErrors = PhpFileLoader::IGNORE_NONE)
```

Comment:
Same as the RouteConfigurator, should be changed in Symfony 5.0.

Reply:
`bool $ignoreErrors` should be changed as we need more values to pass than just true/false. Do you prefer to remove the type check for `int` here?

Reply:
I have removed the type hint. As this method is final, it should be ok.